metadata/upload.go

```diff
@@ -28,6 +29,7 @@ var (
 	projectID       = os.Getenv("GCP_PROJECT")
 	datasetID       = os.Getenv("DATASET_ID")
 	tableID         = getEnv("TABLE_ID", "metadata")
+	gcsFetchBufferSize, _ = strconv.Atoi(os.Getenv("GCS_QUERY_THROTTLE"))
```

Comment:
Please implement proper error handling here, instead of skipping the error check.

Reply:
Also, it looks like if `GCS_QUERY_THROTTLE` is not present by default, this code will fail. We should use some reasonable default value instead. I'll make a suggestion on who to better handle it.